SpatialGDK/Source/SpatialGDK/Private/Interop/ActorSystem.cpp

```diff
@@ -137,37 +137,25 @@ struct ActorSystem::RepStateUpdateHelper
 #endif
 };
 
-struct FSubViewDelta;
+enum class EActorSubViewType
```

Comment:
How is this different from the existing `ENetRole`?

Reply:
1. No ROLE_None/ROLE_Max
2. Actor Subview type isn't necessarily the same as Unreal's Role, even if they do map onto one another.

Reply:
Thanks for the clarification - would it make sense do add a quick comment about this here?